xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-send/xwiki-platform-mail-send-default/src/main/java/org/xwiki/mail/script/ScriptMimeMessage.java

```diff
@@ -79,16 +79,29 @@
         this.execution = execution;
         this.componentManager = componentManager;
     }
+    
+    /**
+     * Add some content to the mail to be sent. Can be called several times to add different content type to the mail.
+     *
+     * @return the Mime Body Part object that was added. Returning it allows script to make modifications to that body
+     *         part after it's been set (get/set some headers, etc)
+     * @param bodyPart a {@link BodyPart} object to include in the mail
+     */
+    public BodyPart addPart(BodyPart bodyPart)
```

Comment:
missing a @since and an @Unstable annotations.

Reply:
@vmassol which version should go in the since tag?

Reply:
`@since 10.0RC1`